bundles/org.openhab.ui.homebuilder/web/package.json

```diff
@@ -12,16 +12,19 @@
   },
   "dependencies": {
     "ascii-table": "0.0.9",
-    "bootstrap-vue": "^0.18.0",
-    "daemonite-material": "^4.0.0-alpha.6",
+    "bootstrap-vue": "~0.18.0",
+    "daemonite-material": "~4.1.1",
```

Comment:
Because of this upgrade we lost some support for 1920 & > resolutions, so it renders smaller then it used to, is that an issue?

Reply:
It was automatically upgraded because of the ^ but decided to fix it on minors of the current version so that we only update it when we intent to do so.

I was in the process of checking out why it was removed when the rest of the family returned from the swimming pool, now I’m outside supervising the smallest one ☺️

Reply:
If no one has real problems with this I prefer to stick with this version, it’s not a bug it’s a feature, see https://github.com/Daemonite/material/issues/161

So if we want to upgrade any time we would face it again, so better just follow the standards now. It scales good on mobile and smaller screens, only for bigger screens it might be a little step down.